src/screens/root/container/rootContainer.js

```diff
@@ -113,11 +116,13 @@ const RootContainer = () => (WrappedComponent) => {
       }
 
       if (profile || content) {
-        navigation.navigate({
-          routeName,
-          params,
-          key: permlink || author,
-        });
+        setTimeout(() => {
```

Comment:
why you added time out here?

Reply:
i suspect, without it redirection won't work properly because app will be still loading up and loose redirect url data if there is no timeout... but please test it without `timeout` as well to confirm if it is necessary or not.

Reply:
yes we should wait until global props loading, we should change all of global props loading structure, I solved with timeout, after we changed global props structure we will remove timeout

Reply:
Can you create GitHub issue for that because we need to make sure that structure is improved, also give context on what should be improved?

Reply:
okay